lib/core/include/irods_default_paths.hpp.in

```diff
@@ -3,8 +3,14 @@
 
 #include <boost/filesystem.hpp>
 
+#define IRODS_DEFAULT_PATH_LIBDIR "@CMAKE_INSTALL_LIBDIR@"
+#define IRODS_DEFAULT_PATH_SYSCONFDIR "@CMAKE_INSTALL_SYSCONFDIR@"
+#define IRODS_DEFAULT_PATH_HOMEDIR "@IRODS_HOME_DIRECTORY@"
+#define IRODS_DEFAULT_PATH_PLUGDIR "@IRODS_PLUGINS_DIRECTORY@"
```

Comment:
unless it's a cmake standard-ish thing...  can we do `PLUGINDIR` instead of `PLUGDIR`?